index.js

```diff
@@ -88,6 +88,7 @@ function Router (opts) {
   this.maxParamLength = opts.maxParamLength || 100
   this.allowUnsafeRegex = opts.allowUnsafeRegex || false
   this.constrainer = new Constrainer(opts.constraints)
+  this.useSemicolonDelimiter = opts.useSemicolonDelimiter || false
```

Comment:
Don't forget to set it `true` in the Fastify.

Reply:
Ideally this option should be disabled in Fastify as well, but I don't know the best way to do it: leave the default `false` here and open issue/PR to update dependencies in Fastify, or make it `true` here and send PR to Fastify with the option disabled

Reply:
I would leave the `false` here, because it's a correct behavior and we can release a new major of fmw at any moment. In current version of Fastify it should be set to `true` by default, because otherwise it's a breaking change.

About changing it to `false` in the fastify in a future release. It's better to open an issue about this in fastify, but IMHO it never would be set to `true` by defauld in the Fastify, because it will break too many apps. It just doesn't worth it even if it's true according to the spec.